devtools/projects/shell-browser/src/app/content-script.ts

```diff
@@ -13,21 +13,35 @@ import {BACKEND_URI, CONTENT_SCRIPT_URI, DETECT_ANGULAR_SCRIPT_URI} from './comm
 import {SamePageMessageBus} from './same-page-message-bus';
 
 let backgroundDisconnected = false;
-let backendInstalled = false;
 let backendInitialized = false;
 
 const port = chrome.runtime.connect({
   name: `${document.title || location.href}`,
 });
 
 const handleDisconnect = (): void => {
-  // console.log('Background disconnected', new Date());
   localMessageBus.emit('shutdown');
   localMessageBus.destroy();
   chromeMessageBus.destroy();
   backgroundDisconnected = true;
 };
 
+function attemptBackendHandshake() {
+  if (!backendInitialized) {
+    // tslint:disable-next-line:no-console
+    console.log('Attempting handshake with backend', new Date());
```

Comment:
Should we use `debug` instead ? 
```suggestion
    console.debug('Attempting handshake with backend', new Date());
```
This will prevent spaming the console by default.

Reply:
I think it's fine to keep it as is. This is logged in the content script context of the DevTools, so it shouldn't be visible to the end users in their app console. Also, ideally, it should be logged once.